x-pack/plugins/grokdebugger/public/register_feature.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  FeatureCatalogueCategory,
+  HomePublicPluginSetup,
+} from '../../../../src/plugins/home/public';
+
+export const registerFeature = (homePlugin: HomePublicPluginSetup) => {
+  homePlugin.featureCatalogue.register({
+    id: 'grokdebugger',
+    title: i18n.translate('xpack.grokDebugger.registryProviderTitle', {
+      defaultMessage: '{grokLogParsingTool} Debugger',
+      values: {
+        grokLogParsingTool: 'Grok',
+      },
+    }),
+    description: i18n.translate('xpack.grokDebugger.registryProviderDescription', {
+      defaultMessage:
+        'Simulate and debug {grokLogParsingTool} patterns for data transformation on ingestion.',
```

Comment:
Same here.

Reply:
This was copy and pasted from the old code, and I can't see why it needs to be treated this way. I'll change it 👍